interface-definitions/vpn-openconnect.xml.in

```diff
@@ -144,6 +154,11 @@
                     </properties>
                     <defaultValue>2</defaultValue>
                   </leafNode>
+                  <leafNode name="groupconfig">
```

Comment:
Maybe `group-config` will be feet better
Do we really need this extra option if we have the `group`? 
```
If the groupconfig option is set, then config-per-user will be overriden, and all configuration will be read from radius
```
Am I understanding correctly that you mean some kind of `authentication mode`
```
vyos@r14# set vpn openconnect authentication mode 
Possible completions:
   local                Use local username/password configuration (OTP supported)
   radius               Use RADIUS server for user autentication

```

Reply:
Yes sir, its a separate function.
You  can define local groups, or RADIUS group checks, but not pull attributes such as IP address _from_ RADIUS itself.
Its why we need to expand the CLI to include per-group configurations (such as the client subnet, routes, NS) so as to permit RADIUS-based user+group auth and device-based assignment of VPN attributes based on the group.